vendor.conf

```diff
@@ -86,7 +86,11 @@ github.com/opencontainers/image-spec                d60099175f88c47cd379c4738d15
 github.com/seccomp/libseccomp-golang                689e3c1541a84461afc49c1c87352a6cedf72e9c # v0.9.1
 
 # systemd integration (journald, daemon/listeners, containerd/cgroups)
+github.com/coreos/go-systemd/v22                    2d78030078ef61b3cae27f42ad6d0e46db51b339 # v22.0.0
+github.com/godbus/dbus/v5                           37bf87eef99d69c4f1d3528bd66e3a87dc201472 # v5.0.3
+# go-systemd v17 is required by github.com/coreos/pkg/capnslog/journald_formatter.go
```

Comment:
Note: dependency on `pkg/caplog` can be removed by updating etcd to master (v3.5) https://github.com/etcd-io/etcd/commit/725e09023ad481c8dc05e2ecce596fb6c7413f5c